q2_feature_table/plugin_setup.py

```diff
@@ -379,7 +383,10 @@
                       'retained.'),
         'prevalence': ('The minimum portion of samples that a feature '
                        'must have a relative abundance of at least '
-                       '`abundance` to be retained.')
+                       '`abundance` to be retained.'),
+        'allow_empty_table': 'If true, the filtered table may be empty. '
+                             'Otherwise, an error will be raised if the '
```

Comment:
```suggestion
                             'Default behavior is to raise an error if the '
```
Same suggestion for all three descriptions.